plugins/PrivacyManager/templates/gdprOverview.twig

```diff
@@ -51,5 +51,28 @@
             <li>Include Matomo in your <a href="https://www.cnil.fr/en/guidelines-dpia" target="_blank" rel="noreferrer noopener">data privacy impact assessment (DPIA)</a>, if applicable.</li>
         </ol>
     </div>
+
+    <div piwik-content-block content-title="Data retention">
+        <p>
+            Data retention for data stored in Matomo:
+        </p>
+        <ul>
+            {% if deleteLogsEnable %}
+                <li>all aggregated reports are deleted after <strong>{{ reportRetention }}</strong>.</li>
+            {% else %}
+                <li>aggregated report are <strong>never</strong> deleted.</li>
+            {% endif %}
+            {% if deleteReportsEnable %}
+                <li>all visitors and actions raw data are deleted after <strong>{{ rawDataRetention }}</strong>.</li>
+            {% else %}
+                <li>visitors and actions raw data are <strong>never</strong> deleted.</li>
```

Comment:
Think it should be "visit and action raw data", since you can't technically delete the visitor themselves :P